conf/igenomes.config

```diff
@@ -258,35 +240,29 @@ params {
             fasta                 = "${params.igenomes_base}/Homo_sapiens/UCSC/hg38/Sequence/WholeGenomeFasta/genome.fa"
             snpeff_db             = 105
             snpeff_genome         = 'GRCh38'
-            snpeff_version        = '5.1'
-            vep_cache_version     = 108
+            vep_cache_version     = 109
             vep_genome            = 'GRCh38'
             vep_species           = 'homo_sapiens'
-            vep_version           = '108.2'
         }
         'hg19' {
             bwa                   = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/BWAIndex/version0.6.0/"
             fasta                 = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/WholeGenomeFasta/genome.fa"
             readme                = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Annotation/README.txt"
             snpeff_db             = 87
             snpeff_genome         = 'GRCh37'
-            snpeff_version        = '5.1'
-            vep_cache_version     = 108
+            vep_cache_version     = 109
             vep_genome            = 'GRCh37'
             vep_species           = 'homo_sapiens'
-            vep_version           = '108.2'
         }
         'mm10' {
             bwa                   = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Sequence/BWAIndex/version0.6.0/"
             fasta                 = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Sequence/WholeGenomeFasta/genome.fa"
             readme                = "${params.igenomes_base}/Mus_musculus/UCSC/mm10/Annotation/README.txt"
             snpeff_db             = 99
             snpeff_genome         = 'GRCm38'
-            snpeff_version        = '5.1'
```

Comment:
love that one of the versions is pulled out

Reply:
I love even more that we're getting rid of the container declaration